content/self-host/pro/_index.en.md

```diff
@@ -24,15 +24,23 @@ RustDesk client >=1.2.0 required
 
 ## Installation
 
-Almost as the same as the open source version, but you do not need to run hbbs/hbbr with any arguments, all can be set later in web console.
+# Simple Install
+
+Copy and paste the above command into your linux terminal which take care of everything.  
+
+`curl -s https://raw.githubusercontent.com/rustdesk/rustdesk-server-pro/main/install.sh | bash`
```

Comment:
More details needed, need to tell users what the script has done clearly.

Reply:
e.g. `WorkingDirectory=/opt/rustdesk/`, `nginx`, `certbot`, how does certbot auto update expiring license? etc.

This can be a seperate page.

Reply:
BTW,  we should use the same working dir.

https://github.com/rustdesk/rustdesk-server/blob/afeebe852d0f4a4bbb2659db37b1a889c0a3a8b5/systemd/rustdesk-hbbs.service#L9

Reply:
> should be used for storage of data,

I prefer /var/lib more than /opt

Reply:
I mean its not right but I dont mind, can change it

Reply:
Right is not very important here.

Reply:
I remember postgres put all things in /var/lib/

Reply:
yeah, the data, anyways just changing them so its consistent

Reply:
https://github.com/rustdesk/rustdesk-server-pro/pull/2

Reply:
https://github.com/rustdesk/rustdesk-server-pro/pull/3
https://github.com/rustdesk/doc.rustdesk.com/pull/216

Reply:
these should both have explanations of what the script does